sapi/fpm/www.conf.in

```diff
@@ -70,7 +70,7 @@ listen = 127.0.0.1:9000
 ; Default Value: no set
 ; process.priority = -19
 
-; Set the process dumpable flag (PR_SET_DUMPABLE prctl) even if the process user
+; Set the process dumpable flag (PR_SET_DUMPABLE prctl for Linux or PROC_TRACE_CTL procctl for FreeBSD) even if the process user
```

Comment:
this line should get wrap as others in the file

Reply:
good point